sme_finance_application_schema/translations.py

```diff
@@ -33,7 +33,8 @@ def finance_application_v3_to_sme_v5(finance_application):
                   'business_plan', 'card_revenue', 'online_revenue', 'institutional_revenue',
                   'stock_ready', 'revenue_growth', 'intellectual_property', 'trade_credit',
                   'business_premises', 'registered_brand', 'customers', 'region',
-                  'company_credit_rating', 'familiarity_with_financing', 'accounting_software'):
+                  'company_credit_rating', 'familiarity_with_financing', 'accounting_software',
```

Comment:
`familiarity_with_financing` is actor level.
as is `directors_pensions`,

Reply:
Bonus points if you find a way to test this kind of bug.
Rough idea being:
```py
class SchemaDict(UserDict):
    __init__(self, schema):
        self.schema = schema
        super()
    __getitem__(self, item):
        if key not in schema: raise
```
But this is a long shot. The Tracer code I wrote for translations will be useful for this.